src/python/dxpy/scripts/dx_build_app.py

```diff
@@ -1014,6 +1014,27 @@ def get_destination_region(destination):
         dest_project_id = dxpy.WORKSPACE_ID
     return dxpy.api.project_describe(dest_project_id, input_params={"fields": {"region": True}})["region"]
 
+def verify_nf_license(destination, extra_args):
+    
+    def get_project_to_check():
+        # extra args overrides the destination argument
+        # so we're checking it first
+        if "project" in extra_args:
+            return extra_args["project"]
+        if destination:
+            dest_project_id, _, _ = parse_destination(destination)
+            return dest_project_id
+        else:
+            return dxpy.PROJECT_CONTEXT_ID
+
+    dest_project_to_check = get_project_to_check()
+    features = dxpy.DXHTTPRequest("/" + dest_project_to_check + "/checkFeatureAccess", {"features": ["dxNextflow"]})['features']
```

Comment:
Shouldnt we have .get('features', {}) to avoid error in case of it missing?

Reply:
@mhrvol I think this feature is already been included with a default value `false` of orgs features. if any name here is invalid we will have `dxpy.exceptions.InvalidInput: Expected key "features" of input to include only valid features.`

but agree that better to use `get` method here

Reply:
Updated, thx!